README.md

```diff
@@ -25,15 +25,20 @@ require "carbon_sendgrid_adapter"
 
 BaseEmail.configure do |settings|
  if LuckyEnv.production?
-   send_grid_key = send_grid_key_from_env
-   settings.adapter = Carbon::SendGridAdapter.new(api_key: send_grid_key)
+   settings.adapter = construct_send_grid_adapter
  else
   settings.adapter = Carbon::DevAdapter.new
  end
 end
 
-private def send_grid_key_from_env
-  ENV["SEND_GRID_KEY"]? || raise_missing_key_message
+private def construct_send_grid_adapter
+  if ENV["SEND_GRID_KEY"]?
+    Carbon::SendGridAdapter.new(api_key: ENV["SEND_GRID_KEY"])
+  elsif ENV["SENDGRID_USER"]? && ENV["SENDGRID_PASSWORD"]?
+    Carbon::SendGridAdapter.new(username: ENV["SENDGRID_USER"], password: ENV["SENDGRID_PASSWORD"])
+  else
+    raise_missing_key_message
```

Comment:
Currently only raises `SEND_GRID_KEY` which is okay I guess? API_KEY should be the default